logevents/src/main/java/org/logevents/observers/ElasticsearchLogEventObserver.java

```diff
@@ -34,34 +40,93 @@
  * observer.elastic.suppressMarkers=PERSONAL_DATA
  * observer.elastic.formatter.excludedMdcKeys=secret
  * </pre>
+ * Optional configuration shown below, but with their defaults
+ * <pre>
+ * observer.elastic.elasticsearchUrlPath=_bulk
+ * </pre>
+ * And you can provide Authorization header with
+ * <pre>
+ * observer.elastic.elasticsearchAuthorizationHeader
+ * </pre>
  */
 public class ElasticsearchLogEventObserver extends AbstractBatchingLogEventObserver {
 
+    public static final String DEFAULT_ELASTICSEARCH_BULK_API_PATH = "_bulk";
     private final URL elasticsearchUrl;
+    protected final String elasticsearchUrlPath;
+    private final String elasticsearchAuthorizationHeaderValue;
     private final String index;
     private final JsonLogEventFormatter formatter;
 
+    private Proxy proxy = Proxy.NO_PROXY;
+
     public ElasticsearchLogEventObserver(Map<String, String> properties, String prefix) {
         this(new Configuration(properties, prefix));
     }
 
     public ElasticsearchLogEventObserver(Configuration configuration) {
         this.elasticsearchUrl = configuration.getUrl("elasticsearchUrl");
+        this.elasticsearchUrlPath = configuration.optionalString("elasticsearchUrlPath").orElse(getDefaultPath());
+        this.elasticsearchAuthorizationHeaderValue = configuration.optionalString("elasticsearchAuthorizationHeader").orElse(NO_AUTHORIZATION_HEADER);
```

Comment:
This is a though one, simply because the name of the observer can be anything, ie `observer.foo.elasticsearchUrlPath` . You can even have `observer.audit.elasticsearch..` , then it is nice that these elasticsearch specific properties has the name elasticsearch in it? 

Maybe it could have been   `configuration.getPojo("elasticsearch", ElasticsearchConfigPojo::class)` and this one eats the `url` and `urlPath` etc. Personally used libraries like https://github.com/lightbend/config etc to this this. However I feel this probably is a bit out of scope for this particular PR, if you don't see a simply way to do this without dragging in libraries.

Reply:
Slept on this, maybe it is actually covered by `observer.audio=HumioLogEventObserver` .. or `observer.audio=ElasticSearchLogEventObserver`?

Maybe a bit weird for Humio to strip the `elasticsearch` prefix since Humio also has another logs injection API: https://library.humio.com/stable/docs/ingesting-data/log-shippers/other-log-shippers/#humio-ingest-api 

They should keep same prefix names due to `HumioLogEventObserver` extending `ElasticSearchLogEventObserver`. Could of course extract and duplicate code or use composition with shared objects or totally seperate em?